src/telliot_feed_examples/reporters/reporter_autopay_utils.py

```diff
@@ -48,46 +48,41 @@ async def get_single_tip(
         msg = "can't suggest single tip for autopay contract not connected"
         error_status(note=msg, log=logger.critical)
         return None
-    query_id_lis, status = await autopay.read("getFundedQueryIds")
+
+    current_time = TimeStamp.now().ts
+
+    tips, status = await autopay.read("getPastTips", _queryId=query_id)
```

Comment:
If we're going to use `getPastTips`, shouldn't we add up all the tips for each unique queryId, because a queryId could have been funded multiple times? And after that, then sort the list by greatest to least tips? And finally, check if no one else has reported for each queryId, checking by descending potential tips?

Reply:
Thats one thing I misunderstood that was explained to me now is that the reporter gets the total amount of tips and not only a single tip from the tips list.  So no need to getPastTips because there is a function that calculates all of that (getCurrentTips(queryId).